### DISCUSSION TITLE
QR Code for app

### DISCUSSION THREAD

**Discussion by rajesh-murali** [NUMde/compass-numapp-frontend#72] [Q&A] [answered]
Dear Team, 
can you please tell how to get a QR code  or participant  codefor this app?

**Comment by mbastian93** [comment, marked as answer]
Use a tool like [this](https://goqr.me/) and enter data in the form of

`{"AppIdentifier":"COMPASS","SubjectId":"7bfc3b07-a97d-4e11-8ac6-b970c1745476"}`

where 'subjectId' can be replaced for each user

**Reply by rajesh-murali** [reply]
thank you. In the developer version how to provide a valid participant code without getting 404 authentication error. I got 404 authentication error.  Where should we include the AppIdentifier subjectId

**Reply by mbastian93** [reply]
Have you set up the backend (and database) locally as described in [that repository](https://github.com/NUMde/compass-numapp-backend)? The subject IDs need to be inserted into the table 'studyparticipant'. There is also [script](https://github.com/NUMde/compass-numapp-backend/tree/main/scripts/createSubjectIDs) which does that.

I hope, that helps you.

**Reply by rajesh-murali** [reply]
thank you. I have not yet set up the backend locally
